jag-crdp-application/src/main/java/ca/bc/gov/open/crdp/models/GuidMapDocument.java

```diff
@@ -1,15 +1,22 @@
 package ca.bc.gov.open.crdp.models;
 
-import java.io.Serializable;
-import java.util.Map;
+import java.util.List;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
-public class GuidMapDocument implements Serializable {
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "GuidMapDocument")
+public class GuidMapDocument {
     private String version;
-    private Map<String, String> mapping;
```

Comment:
is there a reason we have to use self-defined map structure?

Reply:
database side checks if API sends a valid xml file but previously we only sent the map